docs/source/algorithms/SetSample-v1.rst

```diff
@@ -45,6 +45,7 @@ be in centimeters):
 - ``HollowCylinder``: Height, InnerRadius, OuterRadius, Center
 - ``FlatPlateHolder``: Width, Height, Thick, Center, Angle, FrontThick, BackThick. This is a CSG union of 2 FlatPlates tightly wrapping a FlatPlate sample. To be used for the ContainerGeometry.
 - ``HollowCylinderHolder``: Height, InnerRadius, InnerOuterRadius, OuterInnerRadius, OuterRadius, Center. This is a CSG union of 2 HollowCylinders tightly wrapping a HollowCylinder sample. To be used for the ContainerGeometry.
+- ``Sphere``: Center, Radius
```

Comment:
This is a bit lazy of me but I've been meaning to open an issue for another v minor doc change on L44 where the "Axis" property isn't mentioned in here despite being supported in the code (see SetSample::createCylinderLikeXML). Would you mind updating L44 and L45 to add this parameter in eg  `Cylinder: Height, Radius, Center, Axis`?
This came up on a call with the WISH team recently